Profile/__test__/handler.test.ts

```diff
@@ -0,0 +1,77 @@
+import * as E from "fp-ts/lib/Either";
+import { FiscalCode } from "../../generated/definitions/fast-login/FiscalCode";
+import { SpidLevel } from "../../utils/enums/SpidLevels";
+import { IHslJwtPayloadExtended } from "../../utils/middlewares/hsl-jwt-validation-middleware";
+import { profileHandler } from "../handler";
+import { Client } from "../../generated/definitions/io-functions-app/client";
+
+// #region mocks
+const getProfileMock = jest.fn(async () =>
+  E.right({
+    status: 200,
+    value: {
+      email: "example@test.com"
```

Comment:
I think we can put this value inside a variable instead of hardcoding it. we can then reference to it in the tests below more easily